Dalamud/Game/Gui/Nameplates/Model/NameplateObject.cs

```diff
@@ -0,0 +1,13 @@
+namespace Dalamud.Game.Gui.Nameplates.Model;
+
+/// <summary>
+/// Represents a nameplate object (mostly like player or NPC).
+/// </summary>
+/// <param name="pointer">The pointer for the nameplate object.</param>
+public class NameplateObject(IntPtr pointer)
+{
+    /// <summary>
+    /// Gets the pointer for the nameplate object.
+    /// </summary>
+    public IntPtr Pointer { get; } = pointer;
+}
```

Comment:
Per <https://github.com/goatcorp/Dalamud/pull/1808> It'll probably be best to create an interface for this type.